container/root/etc/nginx/sites-available/default

```diff
@@ -44,6 +44,15 @@ server {
     fastcgi_pass 127.0.0.1:9000;
   }
 
+  # Allow locally-accessible (only) FPM status page
+  location ~ ^/__status$ {
+    access_log off;
```

Comment:
we don't want to log this?  are we 100% sure it's limited to local only @bryanlatten ?  the access log would let us verify this.